composer.json

```diff
@@ -5,7 +5,7 @@
     "description": "Bundle that provides controllers and services for using Sulu as headless content management system",
     "require": {
         "php": "^7.3 || ^8.0",
-        "sulu/sulu": "^2.1 || ^2.3@dev",
```

Comment:
the `BlockPropertyType::getSettings` method was added in Sulu 2.2. we could implement a BC layer for this, but that will increase the maintenance effort in the future. also the upgrade from 2.1 to 2.2 is quite easy for projects. therefore i would not add BC layers until there is a stable release for this bundle.